test/unit/css.js

```diff
@@ -1802,6 +1802,7 @@ QUnit.testUnlessIE( "css(--customProperty)", function( assert ) {
 	assert.equal( $elem.css( "--prop10" ), "val10", "Multiple preceding and following escaped unicode whitespace trimmed" );
 	assert.equal( $elem.css( "--prop11" ), "val11", "Multiple preceding and following unicode whitespace trimmed" );
 	assert.equal( $elem.css( "--prop12" ), "\u000Bval12\u000B", "Multiple preceding and following non-CSS whitespace reserved" );
+	assert.equal( $elem.css( "--nonexistent" ), undefined );
```

Comment:
Please also add a test for a CSS variable that is defined but with no meaningful value: e.g. `--empty: ;`. This should also return `undefined`. I wish we were able to return an empty string there but I don't see a way so let's at least test the current behavior.

Reply:
Added.